DefInjected/ResearchProjectDef/ResearchProjects_4_MultiAnalyzer.xml

```diff
@@ -44,7 +44,7 @@
   <!-- EN: powered armor -->
   <PoweredArmor.label>pancerz wspomagany</PoweredArmor.label>
   <!-- EN: Build the power armor and power armor helmets. Note that these also require advanced components. -->
-  <PoweredArmor.description>Neuromemetyczna robotyka i zaawansowane technologie tkackie ze superstali pozwalają zbudować wspomagany pancerz i wspomagane hełmy.</PoweredArmor.description>
+  <PoweredArmor.description>Neuromemetyczna robotyka i zaawansowane technologie tkackie z superstali pozwalają zbudować wspomagany pancerz i wspomagane hełmy.</PoweredArmor.description>
```

Comment:
"superstali" po "s" występuje samogłoska więc prawidłowy zapis to "z superstali"